src/Activation/ActivateProximityTracing.tsx

```diff
@@ -43,7 +43,7 @@ const ActivateProximityTracing: FunctionComponent = () => {
   }
 
   return (
-    <SafeAreaView>
+    <SafeAreaView style={{ backgroundColor: Colors.primaryBackground }}>
```

Comment:
I think I like this pattern more than `style.safeArea` and then specifying `backgroundColor` in the styles. This will only ever take a `backgroundColor` so it feels more straightforward this way.

Reply:
My thoughts on this are that if we extract it to a style key, extending it implies, going to the key and adding the new style, while leaving it here, when it gets extended the person that changes is going to have to move it to the styles(create a variable if it doesn't exist) and then change this line. Also, consistency, if we always have it on the style object there are no opinions, just one pattern to follow, one less thing to consider.

Reply:
Good points! Will update.